wp-content/themes/bpr2018/resources/templates/layout/head.tpl.php

```diff
@@ -170,6 +171,26 @@ class="logo"
             }
           })
         });
+
+        // Lazy load images using Intersection Observer API
+        document.addEventListener("DOMContentLoaded", function() {
+          var lazyBackgrounds = [].slice.call(document.querySelectorAll(".lazy"));
+
+          if ("IntersectionObserver" in window) {
+            let lazyBackgroundObserver = new IntersectionObserver(function(entries, observer) {
```

Comment:
It would be nice to specify a `rootMargin` so that images are loaded just before they are scrolled into view (so extend the bottom detection by something like 200px). When I scroll, the suddenly loading images is a bit jarring.

Reply:
remove var